packages/frontend/lib/pillars.ts

```diff
@@ -1,13 +1,18 @@
 import { PillarColours, palette } from '@guardian/pasteup/palette';
 
-export const pillarNames: Pillar[] = [
+// Pillars are used for styling
+// RealPillars have Pillar palette colours
+// FakePillars allow us to make modifications to style based on rules outside of the pillar of an article
+const realPillars = [
     'news',
     'opinion',
     'sport',
     'culture',
     'lifestyle',
-    'labs',
-];
+] as const;
+const fakePillars = ['labs'] as const;
+
+export const allPillars = [...realPillars, ...fakePillars];
```

Comment:
Could we instead just type these using the types in index.d.ts? E.g. like:

    const allPillars: []Pillar = [..]

Reply:
I'm not sure, @AWare ?